classes/WP_CLI/Action/Get.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * WP-CLI command: action-scheduler action get
+ */
+class ActionScheduler_WPCLI_Action_Get_Command extends ActionScheduler_WPCLI_Command {
+
+	/**
+	 * Execute command.
+	 *
+	 * @return void
+	 */
+	public function execute() {
+		$action_id = $this->args[0];
+		$store     = \ActionScheduler::store();
+		$logger    = \ActionScheduler::logger();
+		$action    = $store->fetch_action( $action_id );
+
+		if ( is_a( $action, ActionScheduler_NullAction::class ) ) {
+			/* translators: %d is action ID. */
+			\WP_CLI::error( sprintf( esc_html__( 'Unable to retrieve action %d.', 'action-scheduler' ), $action_id ) );
+		}
+
+		$only_logs   = ! empty( $this->assoc_args['field'] ) && 'log_entries' === $this->assoc_args['field'];
```

Comment:
Should this also check against `--fields` (ie, if `--fields=log_entries` was passed)?

Reply:
Oh, that's interesting. Why have both `--field` and `--fields` flags? 🤔

Reply:
Sticking to one may be cleaner, and would avoid doubling up on checks here. Instead of supporting [`--field` and `--fields`](https://github.com/woocommerce/action-scheduler/pull/1071/files#diff-ab2b213587b3f26cf90e53ed137c529583b32100193ae7b256a3c58dbb3d2232R184-R188), we could just rely on `--fields`.

Reply:
Sorry, my question was meant for core. Because core implements both, I'm going to keep both.